### DISCUSSION TITLE
Ortogonalidade de operadores

### DISCUSSION THREAD

**Discussion by leonelsanchesdasilva** [DesignLiquido/delegua#616] [Ideias]
Delégua possui um problema herdado de Égua quanto a ortogonalidade de operadores: 

```js
escreva(5 e 2) // 2
escreva(5 & 2) // 0
escreva(5 e 1) // 1
escreva(5 & 1) // 1
```

A meu ver, as comparações deveriam ser equivalentes. Usando o operador `&` o comportamento é idêntico ao do JavaScript (uma comparação bit a bit). Se os comportamentos forem unificados, nem precisamos de símbolos dedicados a operações bit a bit, e símbolos dedicados podem ser simplesmente removidos da linguagem.

**Comment by ItaloCobains** [comment]
Eu concordo esse operador 'e' ou meu ver deixa a sintaxe esquisita.

**Reply by leonelsanchesdasilva** [reply]
@ItaloCobains A verdade a minha ideia é justamente remover o `&` e deixar o `e` no lugar dele.
